site/src/pages/release-notes/1.3.0.mdx

```diff
@@ -0,0 +1,231 @@
+                  SerializationFormat.of("ttuple"), new TTupleProtocol.Factory()));
+      }
+  }
+  ```
+- You can now collect more detailed DNS metrics such as success or failure, query type, etc. #1887, #2935
+- You can now customize the <type://HealthCheckService> when using Spring integration. #3144
+  ```java
+  @Bean
+  public HealthCheckServiceConfigurator healthCheckServiceConfigurator() {
+      return builder -> builder.updatable(true);
+  }
+  ```
+- You can now use <type://RequestHeaders#acceptLanguage()> to choose language. #3177, #3179
+## 📈 Improvements
+
+- Various improvements for Doc service. #3149, #3150, #3167, #3188
+
+## 🛠️ Bug fixes
+- <type://HealthCheckedEndpointGroup#endpoints()> now returns healthy endpoints properly even when
+  <type://EndpointGroup#orElse(EndpointGroup)> is used. #3181
+- `ServletRequest.getProtocol()` now returns the proper value when using <type://TomcatService> and
+  <type://JettyService>. #3194
+- The route decorators are now evaluated in the reverse order they applied. #3160, #3166
+- You now get the FORBIDDEN status if your service does not handle preflight requests regardless of
+  route decorators. #3152
+- A gRPC `ServerCall` is now closed exactly only once. #3153
+- You no longer see `AnnotatedConnectException` when the <type://Endpoint> is created with an
+  IPv6 scope ID. #3158, #3178
+- Armeria server does not reject the request path whose first segment includes a colon anymore. #3154
+
+## 🏚️ Deprecations
+- <type://CircuitBreakerClient#newPerHostAndMethodDecorator(BiFunction,CircuitBreakerRule)> is now deprecated.
+  #31356
+  - Use <type://CircuitBreakerClient#newDecorator(CircuitBreakerMapping,CircuitBreakerRule)> with the customized
+    <type://CircuitBreakerMapping> using <type://CircuitBreakerMapping#builder()>.
+- The response timeout and max total attempts setters in <type://RetryingClientBuilder> is now deprecated.
+  #3128 #3145
+  - The static factory methods that take those parameters in <type://RetryingClient> are now deprecated as well.
+  - Use <type://RetryConfigMapping> and <type://RetryConfig>.
+- <type://Route#apply(RoutingContext)?full> is deprecated. #3152
+  - Use <type://Route#apply(RoutingContext,boolean)?full>.
+- The constructor of <type://PrometheusExpositionService> is now deprecated. #3081
+  - Use <type://PrometheusExpositionService#of(CollectorRegistry)>.
+
+## ☢️ Breaking changes
+
+- N/A
+
+## ⛓ Dependencies
+
+- completalbe-futures 0.3.3 → 0.3.4
+- dropwizard-core 2.0.13 → 2.0.16
+- Fastutil 8.4.2 → 8.4.3
+- gRPC 1.33.0 → 1.33.1
+- grpc-kotlin-stub 0.2.0 → 0.2.1
+- io.dropwizard.metrics 4.1.13 → 4.1.15
+- Jackson 2.11.2 → 2.12.0
+- Jctools 3.1.0 → 3.2.0
+- javax.annotation-api 1.3.2
+- Micrometer 1.5.5 → 1.6.1
+- Netty 4.1.53.Final → 4.1.54.Final
+- org.bouncycastle 1.66 → 1.67
+- Reactor 3.3.10.RELEASE → 3.4.0
+- Spring Boot 2.3.4.RELEASE → 2.4.0
+- Spring 5.2.9.RELEASE → 5.3.1
```

Comment:
You want to add `RELEASE` or not? How about removing them from both sides for consistency?

Reply:
Spring Boot removed `RELEASE` from the artifact name since `2.4.0`.
https://search.maven.org/artifact/org.springframework.boot/spring-boot-starter-jdbc

Reply:
By the way, we specified the Netty version from the artifact version.

Reply:
Can't we just use what they are using?

Reply:
I'm fine with either.